lib/ui/splash_screen/pages/splash_screen.dart

```diff
@@ -0,0 +1,69 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:news_pulse/ui/home/pages/home_parent_page.dart';
+
+import '../../../core/constants/dimens.dart';
+import '../../../core/di/di_manager.dart';
+import '../../../core/utils/ui_utlis/vertical_padding.dart';
+
+class SplashScreen extends StatefulWidget {
+  const SplashScreen({super.key});
+  static const String routeName = "/splash-screen";
+
+  @override
+  State<SplashScreen> createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends State<SplashScreen> {
+  @override
+  void initState() {
+    super.initState();
+    Timer(
+        const Duration(seconds: 1),
+        () => Navigator.pushReplacement(context,
```

Comment:
Don't use this method we already have Navigation system we use and route generator.
To navigate to another screen and you don't want to back to the previous one use the following code 
```dart
DIManager.findNavigator().offAll(Screen.routeName);
```